custom-words.txt

```diff
@@ -608,6 +608,7 @@ cyclicity
 cyrl
 dacl
 daemonset
+daemonsets
```

Comment:
Please update custom-words.txt in a separate PR to main branch.

Reply:
Here is the PR: https://github.com/Azure/azure-rest-api-specs/pull/25533